spec/models/review_spec.rb

```diff
@@ -18,7 +18,8 @@
 
   describe "validations" do
     it { should validate_numericality_of(:rating) }
-    it { should validate_inclusion_of(:rating).in_range(1..5) }
+    it { should validate_inclusion_of(:rating).in_range(1..5).
+                with_message(/ must be between 1 to 5/)}
```

Comment:
Expression at 22, 56 should be on its own line.<br>Use 2 (not 12) spaces for indenting an expression spanning multiple lines.